unattended_scripts/wazuh_install.sh

```diff
@@ -111,6 +111,9 @@ getHelp() {
     echo -e "        -i,  --ignore-health-check"
     echo -e "                Ignores the health-check."
     echo -e ""
+    echo -e "        -s,  --start-cluster"
+    echo -e "                Start the elasticsearch cluster."
```

Comment:
Capitalize Elasticsearch

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/1110/commits/f75864a5d1031e127f0fd67a243eab6d0f5f3c3d